tests/integrations/storage/test_turnoff_titan.rs

```diff
@@ -72,23 +72,36 @@ fn test_turnoff_titan() {
         assert!(db.set_options_cf(handle, &opt).is_ok());
     }
     cluster.compact_data();
-    // wait for gc completes.
-    sleep_ms(10);
-    for i in cluster.get_node_ids().into_iter() {
-        let db = cluster.get_engine(i);
-        assert_eq!(
-            db.get_property_int(&"rocksdb.num-files-at-level0").unwrap(),
-            0
-        );
-        assert_eq!(
-            db.get_property_int(&"rocksdb.num-files-at-level1").unwrap(),
-            1
-        );
-        assert_eq!(
-            db.get_property_int(&"rocksdb.titandb.num-live-blob-file")
-                .unwrap(),
-            0
-        );
+    let mut all_check_pass = true;
+    for _ in 0..10 {
+        // wait for gc completes.
+        sleep_ms(10);
+        all_check_pass = true;
+        for i in cluster.get_node_ids().into_iter() {
+            let db = cluster.get_engine(i);
+            if db.get_property_int(&"rocksdb.num-files-at-level0").unwrap() > 0 {
+                all_check_pass = false;
+                break;
+            }
+            if db.get_property_int(&"rocksdb.num-files-at-level1").unwrap() > 1 {
```

Comment:
Is it correct that all three properties are 0?

Reply:
/cc @tabokie

Reply:
nope, should be exact 1.